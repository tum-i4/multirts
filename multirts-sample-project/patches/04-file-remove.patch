diff --git a/multirts-sample-project/baz/foobar/src/test/resources/test.txt b/multirts-sample-project/baz/foobar/src/test/resources/test.txt
deleted file mode 100644
index f6ea049..0000000
--- a/multirts-sample-project/baz/foobar/src/test/resources/test.txt
+++ /dev/null
@@ -1 +0,0 @@
-foobar
\ No newline at end of file
