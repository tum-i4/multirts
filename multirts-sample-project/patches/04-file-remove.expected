module-selection:
    modules:

test-selection:
    tests:
        a.b.c.foobar.FileOpenTest

    modules:
        baz/foobar/pom.xml